mutacc/cli/importing.py

```diff
@@ -0,0 +1,56 @@
+import json
+import sys, os
+
+import click
+
+from mutacc.utils.hk_call import find_files
+
+@click.command('import')
+@click.option('--bam', multiple = True, type = click.Path(exists = True), help = 'path to .bam file')
+@click.option('--fastq', multiple = True, type = click.Path(exists = True), help = 'path to .fastq file')
+@click.option('--variant', help = 'Variant data as JSON string, or path to .json file')
+@click.option('--case', help = 'Metadata for the case as JSON string or path to .json file')
+@click.pass_context
+def importing(context,bam,fastq,variant,case):
+
+    """
+        Import cases to the database
+    """
+            
+    case = parse_JSON(case)
```

Comment:
Nice! You might want to consider adding this to a "parse" package and import it to the Cli.

Reply:
convention is to use small letters only in function names.
Classes start with big letter.